test/scripts/e2e-cp.sh

```diff
@@ -138,6 +146,11 @@ if [ "$ENABLE_ROUTE_ADVERTISEMENTS" != true ]; then
   skip $BGP_TESTS
 else
   if [ "$ADVERTISE_DEFAULT_NETWORK" = true ]; then
+    # Filter out extended RouteAdvertisements tests to keep job run time down
+    if [ "$ENABLE_NETWORK_SEGMENTATION" = true ]; then
+      skip_label "Feature:RouteAdvertisements && EXTENDED"
```

Comment:
Why are EXTENDED skipped only for BGP? Shouldn't this be universal?
I know that this is only used for BGP e2es right now but I don't think this is specific to the feature in general.
We can address it as a followup.